docs/controls/risk_assessment/ra-05.md

```diff
@@ -0,0 +1,31 @@
+# Risk Assessment
+## RA-05 - VULNERABILITY SCANNING
+
+The organization:  
+a. Scans for vulnerabilities in the information system and hosted applications [Assignment: organization-defined frequency and/or randomly in accordance with organization-defined process] and when new vulnerabilities potentially affecting the system/applications are identified and reported;  
+
+b. Employs vulnerability scanning tools and techniques that facilitate interoperability among tools and automate parts of the vulnerability management process by using standards for:  
+   1. Enumerating platforms, software flaws, and improper configurations;  
+   2. Formatting checklists and test procedures; and  
+   3. Measuring vulnerability impact;  
+
+c. Analyzes vulnerability scan reports and results from security control assessments;  
+
+d. Remediates legitimate vulnerabilities [Assignment: organization-defined response times] in accordance with an organizational assessment of risk; and  
+
+e. Shares information obtained from the vulnerability scanning process and security control assessments with [Assignment: organization-defined personnel or roles] to help eliminate similar vulnerabilities in other information systems (i.e., systemic weaknesses or deficiencies).  
+
+### TDP Implementation
+
+a. As part of the TDP Test Plan, security scans are completed on an ongoing basis, throughout the Continuous Integration (CI).  Automated scans are run on every checkin.  
+
+b. Security scanning is completed using OWASP ZAP dynamic security scans and Snyk vulnerability dependency scanning.  Snyk will automatically open Pull Requests if there is a vulnerability dependency.  If there are no findings, no Pull Requests will be opened.  OWASP Zap scans are the last step for each CI run.  The results for the scans are summarized and accessed through CircleCI.  
+
+c. Summaries of the security scan reports are reviewed in CircleCI.  
```

Comment:
would be helpful to include a screenshot?

Reply:
Yes, this would be good.  I've never accessed CircleCI before.  I could grab the screenshot if I could get access.

Reply:
Added screenshot to implementation statement